unison-src/transcripts/generic-parse-errors.output.md

```diff
@@ -30,12 +30,32 @@ namespace.blah = 1
 
   Loading changes detected in scratch.u.
 
-  The identifier `namespace` used here is a reserved keyword: 
+  I got confused here:
```

Comment:
This regression is weird... I don't know why the message is so much worse now. The only change I made to the lexer was adding "namespace" to the list of keywordy things here:

https://github.com/unisonweb/unison/pull/5285/files#diff-06c08c644fa09b2d2218485a64c9965261fb6bf70d88cf0faf62c1cbe16443f2R576

"namespace" was already partially considered a keyword because it's in this set: https://github.com/unisonweb/unison/blob/f2adc77e39d72d5125f7b2f0a43279aa26119588/unison-syntax/src/Unison/Syntax/ReservedWords.hs#L27

Have I done the wrong thing here?

Reply:
What happens when you use a different reserved word as a name segment?

Reply:
Maybe @sellout would have an idea too?

Reply:
Disregard, I guess this is superseded by https://github.com/unisonweb/unison/pull/5285#issuecomment-2294455794.